components/FilterBar/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { RiArrowDropDownLine } from 'react-icons/ri';
+import Filter from '../../types/helper';
+import * as styles from './styles';
```

Comment:
instead of import * as styles, can you import each individual styled component? It should look like` import {filterBarStyles, filterButtonStyles} from './styles';`